lighthouse-core/lib/minification-estimator.js

```diff
@@ -50,6 +50,12 @@ function computeTokenLength(content, features) {
   let isInRegexCharacterClass = false;
   let stringOpenChar = null;
 
+  /**
+   * Acts as stack for brace tracking.  Entries are true when innermost brace is template literal.
+   * @type boolean[]
```

Comment:
```suggestion
   * @type {boolean[]}
```

(not a nit: without the braces, types-in-jsdoc doesn't work)